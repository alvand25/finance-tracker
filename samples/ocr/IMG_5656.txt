CNY? y \ ih
46-02 Qucens D1vd,
' 5unnyaide, NY 11104
(718) 706-6563
CILANTRO MACHO 2.49 F
BA5IL 2.50 F
TAX 0.00
: anette BALANCE 4.99 a"
HHH HHI KK EERIE ER I OEE
. MasterCard Card - CONTACTLE55
: ACCOUNT NUMBER: *#*#%%xxxx#*2836
APPROVAL CODE: 81146P
5EQUENCE NUMBER: 3310
TERMINAL ID:
TOTAL AMOUNT: 4.99 Purchose
RE5PON5E CODE: APPROVED
04/11/25 04:49pm 110 3
FGI GIT5IGGIUJUI UGLIER He
, MasterCard 4.99
CHANGE 0.00
TOTNL NUMBER OF ITEM5 5ULU - 2
04/11/25 04:49pm 14948 3 376 11o
Jain the 5avinos C1ub for
Additiona1 Do1tor 5avinas.
Thank You for shopping with ws
Your couhter was
Forchosh con/Prochmave vig. di1
ect Tuttter con/Frestia