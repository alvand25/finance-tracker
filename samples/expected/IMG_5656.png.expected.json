{
  "store": "H_Mart",
  "items": [
    {
      "description": "PRODUCE",
      "price": 3.36
    },
    {
      "description": "MILK PAN BREAD",
      "price": 6.95
    },
    {
      "description": "BEAN SPROUTS/12oz",
      "price": 1.99
    },
    {
      "description": "ORG BEECH MUSHROOM",
      "price": 3.99
    },
    {
      "description": "MASAMAN CURRY PASTE",
      "price": 1.99
    },
    {
      "description": "GREEN CURRY PASTE",
      "price": 1.99
    },
    {
      "description": "HS PICKLED RADISH",
      "price": 1.49
    },
    {
      "description": "LIME",
      "price": 0.50
    },
    {
      "description": "PREMIUM BEEF RIBEYE",
      "price": 8.90
    },
    {
      "description": "RICE STICK NOODLES",
      "price": 2.99
    }
  ],
  "subtotal": 34.15,
  "tax": 0.0,
  "total": 34.15,
  "payment_method": "MasterCard",
  "timestamp": "2024-04-12T18:45:00",
  "check_items_exactly": true
}